modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/LayoutProvider/LayoutProvider.es.js

```diff
@@ -201,12 +200,10 @@ class LayoutProvider extends Component {
 						defaultLanguageId,
 						editingLanguageId
 					),
-					hovered: fieldHovered.fieldName === field.fieldName,
```

Comment:
Due to delegate pattern in the old implementation We were handling the Actions shell on LayoutProvider. It creates more complexity on LayoutProvider